src/hotspot/cpu/x86/macroAssembler_x86.cpp

```diff
@@ -930,7 +930,7 @@ void MacroAssembler::long_move(VMRegPair src, VMRegPair dst, Register tmp, int i
       }
     } else {
       assert(dst.is_single_reg(), "not a stack pair: (%s, %s), (%s, %s)",
-       src.first()->name(), src.second()->name(), dst.first()->name(), dst.second()->name());
+        src.first()->name(), src.second()->name(), dst.first()->name(), dst.second()->name());
```

Comment:
Still not indented properly.

Reply:
Shouldn't there be a 2-space indentation wrt the assert here? I could also indent all the arguments to be aligned with the format string, if that seems better.

Reply:
It's preferred to indent multi-line argument lists on the column where argument list starts.

Reply:
```
assert(dst.is_single_reg(), "not a stack pair: (%s, %s), (%s, %s)",
       src.first()->name(), src.second()->name(), dst.first()->name(), dst.second()->name());
```

Reply:
Done